### ISSUE TITLE
Customize date format

### ISSUE THREAD

**Issue by krillin666** [noembryo/KoHighlights#22]
Would it be possible to customize the format of the date ? For example, to something like:
```
Page 27 [[Jul 19th, 2022]]:
Mackay had a low opinion of all Crusades. The Children’s Crusade struck him as only slightly more sordid than the ten Crusades for 

```
The double brackets would be to better integrate the highlights with Logseq.
Thank you !

**Comment by noembryo** [comment]
The part with the customization of the date, might be possible if I also find a way to integrate the setting to the GUI without creating a separate settings dialog, but it will have to wait awhile since I'm on holidays..

The meaning of the Logseq part OTOH, it's a mystery to me..

**Comment by noembryo** [comment]
Fixed in 1.5.0.0
